config/management/src/storage_helper.rs

```diff
@@ -19,6 +19,7 @@ pub struct StorageHelper {
     temppath: libra_temppath::TempPath,
 }
 
+#[allow(dead_code)]
```

Comment:
why keep things around, what is dead?

Reply:
I actually want to resolve this one before landing ... maybe just delete what we don't use? Unless you did something really sneaky and I didn't notice it :P